add-item.html

```diff
@@ -2,15 +2,228 @@
+                                <option value="30k">30k</option>
+                                <option value="40k">50k</option>
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="other__buttons">
+                        <div class="radio">
+                            <h3>What identifies you the best?</h3>
+                            <div class="radio__buttons">
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button" value="asian" required />
+                                    <label for="radio__button">Asian</label>
+                                </div>
+
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button" value="hispanic" required />
+                                    <label for="radio__button">Hispanic</label>
+                                </div>
+
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button" value="african" required />
+                                    <label for="radio__button">African</label>
+                                </div>
+
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button" value="european" required />
+                                    <label for="radio__button">European</label>
+                                </div>
+
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button" value="latina" required />
+                                    <label for="radio__button">Latin American</label>
+                                </div>
+
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button" value="other" required />
+                                    <label for="radio__button">Other</label>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="checkbox">
+                            <h3>Choose everything that applies.</h3>
+
+                            <div class="checkbox__options">
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="student" />
+                                    <label for="checkbox__option">Student</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option" value="teacher" />
```

Comment:
try to use unique id for unique elements.
And see how to use label:for attriubute with input elements.
In this case, whichever checkbox label I click , It will select same checkbox item